### DISCUSSION TITLE
Laragon Cronical

### DISCUSSION THREAD

**Discussion by StefanoTesla** [leokhoa/laragon#882] [General]
Hello I have a question about Cronical setup, currently I'm running every minute a .bat file that runs the command `php artisan schedule:run` to be sure to run scheduler, is the right way or in this way I'm just creating a new instance of the command?

How can I check if the command still alive? and if not run again?

This is mine jobs in the Cronical.dat:
```
# --- Jobs --------------------------------------------------------------------------
*    *    * * *       C:\laragon\www\ascomconnector\runschedule.bat
```

this is the .bat file inside the main laravel folder:

`php artisan schedule:run`

Another thing I don't understand is how get the messages in the cronical log, and I got another problem debuggin Cronical:

If I perform `Cronical.exe --debug` I get an windows error message that tell me Cronical should be installed as a Service, but I already done it, how can I see the process are running?

I have senn in the cronical.dat these type of command:

`@reboot                cmd /c echo Run when the service is started (typically on reboot)`
`@service               cmd /k echo Start and keep running, restarting if it exits`

probably `@service` is what I need but why should restart if it exist? There's something like "keep alive and restart if is not running"?

**Comment by Pen-y-Fan** [comment]
Possibly my blog will help you to understand the concept of running conical [How to run cron jobs on Windows](https://pen-y-fan.github.io/2023/01/28/how-to-run-cron-jobs-on-windows/)
